Lib/test/test_dbm_gnu.py

```diff
@@ -72,17 +72,21 @@ def test_reorganize(self):
         self.g = gdbm.open(filename, 'c')
         size0 = os.path.getsize(filename)
 
-        self.g['x'] = 'x' * 10000
+        # bpo-33901: on macOS with gdbm 1.15, an empty database uses 16 MiB
+        # and adding an entry of 10,000 B has no effect on the file size.
+        # Add size0//2 bytes to make sure that the file size changes.
+        value_size = max(size0 // 2, 10000)
```

Comment:
Why not `size0`? This will guarantee that the file size will be increased (unless the database uses a compression, that is unlikely).

Reply:
I modified my PR to use size0.